common/src/main/java/io/druid/math/expr/Function.java

```diff
@@ -834,7 +835,7 @@ public ExprEval apply(List<Expr> args, Expr.ObjectBinding bindings)
       }
```

Comment:
Above, `DateTimes.wrapFormatter(ISODateTimeFormat.dateOptionalTimeParser())` could be cached more globally.

Reply:
:+1: